.github/workflows/ci.yml

```
@@ -208,10 +208,12 @@ jobs:
 
       - name: Run smoke tests on 'rw dev', 'rw serve', 'rw storybook'
         working-directory: ./tasks/smoke-test
-        run: npx playwright test
+        run: npx playwright test --project replay-chromium --reporter @replayio/playwright/reporter,line
```

```suggestion
        run: npx playwright test --project ${{ matrix.os == 'ubuntu-latest' && 'replay-chromium' || 'chromium' }} --reporter @replayio/playwright/reporter,line
```

Replay's chromium fork isn't supported on windows yet so swapping out the project for windows is probably the simplest solution. You could run with Replay's Firefox fork on windows but I suspect you'll want to stick with the same browser family on each platform.